soccer/motion/MotionControlNode.cpp

```diff
@@ -23,7 +23,7 @@ void MotionControlNode::runMotion(
         MotionControl& controller = _controllers[i];
         MotionSetpoint* setpoint = &(*setpoints)[i];
         if (force_stop) {
-            controller.stop(setpoint);
+            MotionControl::stop(setpoint);
```

Comment:
I'm not a fan of this one, but it's hard to find reasons besides it doesn't look right in terms of the data flow within this function.

Reply:
Yeah... currently I don't think the `MotionControl::stop` function makes sense. Maybe it makes more sense to have a `setpoint.zero()` function or something?